omegaconf/basecontainer.py

```diff
@@ -471,7 +482,14 @@ def _list_merge(dest: Any, src: Any) -> None:
                 for item in src._iter_ex(resolve=False):
                     temp_target.append(item)
 
-            dest.__dict__["_content"] = temp_target.__dict__["_content"]
+            if list_merge_mode == "EXTEND":
+                dest.__dict__["_content"].extend(temp_target.__dict__["_content"])
+            elif list_merge_mode == "EXTEND_IGNORE_DUPLICATES":
```

Comment:
An Enum would also allow you to document this in a discoverable place.

Reply:
Could you elaborate on that? Should I add another code comment?